examples/dask_simple.py

```diff
@@ -0,0 +1,47 @@
+"""
+Optuna example that runs optimization trials in parallel using Dask.
+
+In this example, we perform hyperparameter optimization on a
+RandomForestClassifier which is trained using the Iris dataset.
+Trials are run in parallel on a Dask cluster using Optuna's
+DaskStorage integration.
+
+To run this example:
+
+    $ python dask_simple.py
+"""
+
+from dask.distributed import Client
+import joblib
+from sklearn.datasets import load_iris
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+
+import optuna
+
+
+def objective(trial):
+    X, y = load_iris(return_X_y=True)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=2)
+
+    max_depth = trial.suggest_int("max_depth", 2, 10)
+    n_estimators = trial.suggest_int("n_estimators", 1, 100)
+
+    clf = RandomForestClassifier(max_depth=max_depth, n_estimators=n_estimators)
+    clf.fit(X_train, y_train)
+    y_pred = clf.predict(X_test)
+    score = accuracy_score(y_test, y_pred)
+    return score
```

Comment:
I suspect this is because of how small the iris dataset is. I updated the example to use a [larger dataset](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_digits.html) and now it takes several iterations to find the best trial. Below is the output from running the updated version of this example

<details>
<summary>Details:</summary>

```
Dask dashboard is available at http://127.0.0.1:8787/status
examples/dask_simple.py:41: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  storage = optuna.integration.dask.DaskStorage()
[I 2021-07-22 17:40:06,912] A new study created in memory with name: no-name-e485b064-1e1a-4e32-b0ab-b1706c2ab60e
/Users/james/projects/optuna/optuna/optuna/study/study.py:1156: ExperimentalWarning: DaskStudy is experimental (supported from v2.9.0). The interface can change in the future.
  study = study_class(study_name=study_name, storage=storage, sampler=sampler, pruner=pruner)
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:07,315] Trial 0 finished with value: 0.9333333333333333 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 0 with value: 0.9333333333333333.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:07,337] Trial 1 finished with value: 0.9266666666666666 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 0 with value: 0.9333333333333333.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:07,354] Trial 4 finished with value: 0.9222222222222223 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 6 with value: 0.94.
[I 2021-07-22 17:40:07,355] Trial 6 finished with value: 0.94 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 6 with value: 0.9422222222222222.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:07,378] Trial 5 finished with value: 0.9333333333333333 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 3 with value: 0.9422222222222222.
[I 2021-07-22 17:40:07,384] Trial 7 finished with value: 0.9333333333333333 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 3 with value: 0.9422222222222222.
[I 2021-07-22 17:40:07,402] Trial 2 finished with value: 0.9333333333333333 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 3 with value: 0.9422222222222222.
[I 2021-07-22 17:40:07,418] Trial 3 finished with value: 0.9422222222222222 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 3 with value: 0.9422222222222222.
[I 2021-07-22 17:40:07,645] Trial 8 finished with value: 0.9288888888888889 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 3 with value: 0.9422222222222222.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:07,680] Trial 9 finished with value: 0.9444444444444444 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 10 with value: 0.9466666666666667.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:07,685] Trial 10 finished with value: 0.9466666666666667 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 10 with value: 0.9466666666666667.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:07,693][I 2021-07-22 17:40:07,693] Trial 11 finished with value: 0.9333333333333333 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 10 with value: 0.9466666666666667.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
 Trial 14 finished with value: 0.9288888888888889 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 9 with value: 0.9466666666666667.
[I 2021-07-22 17:40:07,725] Trial 12 finished with value: 0.94 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 10 with value: 0.9466666666666667.
[I 2021-07-22 17:40:07,767] Trial 13 finished with value: 0.9288888888888889 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 10 with value: 0.9466666666666667.
[I 2021-07-22 17:40:07,789] Trial 15 finished with value: 0.9266666666666666 and parameters: {'max_depth': 8, 'n_estimators': 11}. Best is trial 10 with value: 0.9466666666666667.
[I 2021-07-22 17:40:08,108] Trial 16 finished with value: 0.9133333333333333 and parameters: {'max_depth': 5, 'n_estimators': 35}. Best is trial 10 with value: 0.9466666666666667.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:08,156] Trial 17 finished with value: 0.9266666666666666 and parameters: {'max_depth': 5, 'n_estimators': 35}. Best is trial 10 with value: 0.9466666666666667.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:08,168] Trial 19 finished with value: 0.9155555555555556 and parameters: {'max_depth': 5, 'n_estimators': 35}. Best is trial 10 with value: 0.9466666666666667.
[I 2021-07-22 17:40:08,169] Trial 18 finished with value: 0.9133333333333333 and parameters: {'max_depth': 5, 'n_estimators': 35}. Best is trial 10 with value: 0.9466666666666667.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:08,204] Trial 21 finished with value: 0.9066666666666666 and parameters: {'max_depth': 5, 'n_estimators': 35}. Best is trial 10 with value: 0.9466666666666667.[I 2021-07-22 17:40:08,204] Trial 20 finished with value: 0.9133333333333333 and parameters: {'max_depth': 5, 'n_estimators': 35}. Best is trial 10 with value: 0.9466666666666667.

[I 2021-07-22 17:40:08,256] Trial 23 finished with value: 0.9022222222222223 and parameters: {'max_depth': 5, 'n_estimators': 35}. Best is trial 10 with value: 0.9466666666666667.
[I 2021-07-22 17:40:08,269] Trial 22 finished with value: 0.8955555555555555 and parameters: {'max_depth': 5, 'n_estimators': 35}. Best is trial 10 with value: 0.9466666666666667.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:08,897] Trial 24 finished with value: 0.9555555555555556 and parameters: {'max_depth': 10, 'n_estimators': 91}. Best is trial 24 with value: 0.9555555555555556.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:08,929] Trial 28 finished with value: 0.9488888888888889 and parameters: {'max_depth': 10, 'n_estimators': 91}. Best is trial 24 with value: 0.9555555555555556.
[I 2021-07-22 17:40:08,956] Trial 26 finished with value: 0.9533333333333334 and parameters: {'max_depth': 10, 'n_estimators': 91}. Best is trial 29 with value: 0.9577777777777777.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:08,964] Trial 29 finished with value: 0.9577777777777777 and parameters: {'max_depth': 10, 'n_estimators': 91}. Best is trial 29 with value: 0.9577777777777777.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:08,986] Trial 25 finished with value: 0.9577777777777777 and parameters: {'max_depth': 10, 'n_estimators': 91}. Best is trial 25 with value: 0.9577777777777777.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:09,036] Trial 30 finished with value: 0.9555555555555556 and parameters: {'max_depth': 10, 'n_estimators': 91}. Best is trial 25 with value: 0.9622222222222222.
[I 2021-07-22 17:40:09,070] Trial 27 finished with value: 0.9622222222222222 and parameters: {'max_depth': 10, 'n_estimators': 91}. Best is trial 27 with value: 0.9622222222222222.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:09,138] Trial 31 finished with value: 0.9555555555555556 and parameters: {'max_depth': 10, 'n_estimators': 91}. Best is trial 27 with value: 0.9622222222222222.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:09,594] Trial 32 finished with value: 0.9511111111111111 and parameters: {'max_depth': 9, 'n_estimators': 81}. Best is trial 27 with value: 0.9622222222222222.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:09,624] Trial 33 finished with value: 0.9488888888888889 and parameters: {'max_depth': 9, 'n_estimators': 81}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:09,641] Trial 34 finished with value: 0.9488888888888889 and parameters: {'max_depth': 9, 'n_estimators': 81}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:09,656] Trial 36 finished with value: 0.9622222222222222 and parameters: {'max_depth': 9, 'n_estimators': 81}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:09,673] Trial 35 finished with value: 0.9644444444444444 and parameters: {'max_depth': 9, 'n_estimators': 81}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:09,719] Trial 37 finished with value: 0.96 and parameters: {'max_depth': 9, 'n_estimators': 81}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:09,759] Trial 38 finished with value: 0.9555555555555556 and parameters: {'max_depth': 9, 'n_estimators': 81}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:09,818] Trial 39 finished with value: 0.9555555555555556 and parameters: {'max_depth': 9, 'n_estimators': 81}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:10,167] Trial 40 finished with value: 0.8666666666666667 and parameters: {'max_depth': 3, 'n_estimators': 70}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:10,203] Trial 41 finished with value: 0.8822222222222222 and parameters: {'max_depth': 3, 'n_estimators': 70}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:10,206] Trial 44 finished with value: 0.8511111111111112 and parameters: {'max_depth': 3, 'n_estimators': 70}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:10,216] Trial 42 finished with value: 0.86 and parameters: {'max_depth': 3, 'n_estimators': 70}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:10,226] Trial 43 finished with value: 0.86 and parameters: {'max_depth': 3, 'n_estimators': 70}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:10,291] Trial 45 finished with value: 0.8488888888888889 and parameters: {'max_depth': 3, 'n_estimators': 70}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:10,319] Trial 46 finished with value: 0.8555555555555555 and parameters: {'max_depth': 3, 'n_estimators': 70}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:10,420] Trial 47 finished with value: 0.8666666666666667 and parameters: {'max_depth': 3, 'n_estimators': 70}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:10,961] Trial 48 finished with value: 0.9488888888888889 and parameters: {'max_depth': 7, 'n_estimators': 100}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:11,025] Trial 49 finished with value: 0.9377777777777778 and parameters: {'max_depth': 7, 'n_estimators': 100}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:11,031] Trial 50 finished with value: 0.9422222222222222 and parameters: {'max_depth': 7, 'n_estimators': 100}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:11,054] Trial 51 finished with value: 0.94 and parameters: {'max_depth': 7, 'n_estimators': 100}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:11,094] Trial 52 finished with value: 0.9422222222222222 and parameters: {'max_depth': 7, 'n_estimators': 100}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:11,125] Trial 53 finished with value: 0.9466666666666667 and parameters: {'max_depth': 7, 'n_estimators': 100}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:11,157] Trial 54 finished with value: 0.94 and parameters: {'max_depth': 7, 'n_estimators': 100}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:11,263] Trial 55 finished with value: 0.9466666666666667 and parameters: {'max_depth': 7, 'n_estimators': 100}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:11,785] Trial 56 finished with value: 0.96 and parameters: {'max_depth': 9, 'n_estimators': 86}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:11,891] Trial 57 finished with value: 0.96 and parameters: {'max_depth': 9, 'n_estimators': 86}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:11,927] Trial 58 finished with value: 0.9577777777777777 and parameters: {'max_depth': 9, 'n_estimators': 86}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:11,939] Trial 59 finished with value: 0.9533333333333334 and parameters: {'max_depth': 9, 'n_estimators': 86}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:12,032] Trial 62 finished with value: 0.9533333333333334 and parameters: {'max_depth': 9, 'n_estimators': 86}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:12,057] Trial 61 finished with value: 0.9533333333333334 and parameters: {'max_depth': 9, 'n_estimators': 86}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:12,132] Trial 60 finished with value: 0.9555555555555556 and parameters: {'max_depth': 9, 'n_estimators': 86}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:12,274] Trial 63 finished with value: 0.9644444444444444 and parameters: {'max_depth': 9, 'n_estimators': 86}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:12,817] Trial 64 finished with value: 0.9644444444444444 and parameters: {'max_depth': 9, 'n_estimators': 75}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:12,871] Trial 65 finished with value: 0.9533333333333334 and parameters: {'max_depth': 10, 'n_estimators': 75}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:12,939] Trial 68 finished with value: 0.9622222222222222 and parameters: {'max_depth': 10, 'n_estimators': 75}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:12,945] Trial 66 finished with value: 0.9577777777777777 and parameters: {'max_depth': 10, 'n_estimators': 75}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:13,001] Trial 69 finished with value: 0.9577777777777777 and parameters: {'max_depth': 10, 'n_estimators': 75}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:13,089] Trial 67 finished with value: 0.9555555555555556 and parameters: {'max_depth': 10, 'n_estimators': 75}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:13,108] Trial 70 finished with value: 0.9622222222222222 and parameters: {'max_depth': 10, 'n_estimators': 75}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:13,221] Trial 71 finished with value: 0.96 and parameters: {'max_depth': 10, 'n_estimators': 75}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:13,557] Trial 72 finished with value: 0.9377777777777778 and parameters: {'max_depth': 8, 'n_estimators': 57}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:13,622] Trial 73 finished with value: 0.9422222222222222 and parameters: {'max_depth': 8, 'n_estimators': 60}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:13,694] Trial 75 finished with value: 0.9488888888888889 and parameters: {'max_depth': 8, 'n_estimators': 60}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:13,703] Trial 74 finished with value: 0.9444444444444444 and parameters: {'max_depth': 8, 'n_estimators': 60}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:13,783] Trial 76 finished with value: 0.9488888888888889 and parameters: {'max_depth': 8, 'n_estimators': 60}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:13,845] Trial 77 finished with value: 0.9466666666666667 and parameters: {'max_depth': 8, 'n_estimators': 60}. Best is trial 35 with value: 0.9644444444444444.
[I 2021-07-22 17:40:13,872] Trial 78 finished with value: 0.9466666666666667 and parameters: {'max_depth': 8, 'n_estimators': 60}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:13,984] Trial 79 finished with value: 0.9444444444444444 and parameters: {'max_depth': 8, 'n_estimators': 60}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:14,407] Trial 80 finished with value: 0.9577777777777777 and parameters: {'max_depth': 10, 'n_estimators': 78}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:14,468] Trial 81 finished with value: 0.9488888888888889 and parameters: {'max_depth': 10, 'n_estimators': 78}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:14,611] Trial 82 finished with value: 0.9577777777777777 and parameters: {'max_depth': 10, 'n_estimators': 95}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:14,634] Trial 83 finished with value: 0.9577777777777777 and parameters: {'max_depth': 10, 'n_estimators': 95}. Best is trial 35 with value: 0.9644444444444444.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:14,691] Trial 85 finished with value: 0.96 and parameters: {'max_depth': 10, 'n_estimators': 95}. Best is trial 84 with value: 0.9688888888888889.
[I 2021-07-22 17:40:14,739] Trial 84 finished with value: 0.9688888888888889 and parameters: {'max_depth': 10, 'n_estimators': 95}. Best is trial 84 with value: 0.9688888888888889.
[I 2021-07-22 17:40:14,781] Trial 86 finished with value: 0.9577777777777777 and parameters: {'max_depth': 10, 'n_estimators': 95}. Best is trial 84 with value: 0.9688888888888889.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:14,856] Trial 87 finished with value: 0.9533333333333334 and parameters: {'max_depth': 10, 'n_estimators': 95}. Best is trial 84 with value: 0.9688888888888889.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:15,262] Trial 89 finished with value: 0.9533333333333334 and parameters: {'max_depth': 9, 'n_estimators': 78}. Best is trial 84 with value: 0.9688888888888889.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:15,288] Trial 88 finished with value: 0.9511111111111111 and parameters: {'max_depth': 9, 'n_estimators': 95}. Best is trial 84 with value: 0.9688888888888889.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:15,424] Trial 90 finished with value: 0.9555555555555556 and parameters: {'max_depth': 9, 'n_estimators': 83}. Best is trial 84 with value: 0.9688888888888889.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:15,453] Trial 91 finished with value: 0.9622222222222222 and parameters: {'max_depth': 9, 'n_estimators': 83}. Best is trial 84 with value: 0.9688888888888889.
/Users/james/projects/dask/distributed/distributed/protocol/pickle.py:75: ExperimentalWarning: DaskStorage is experimental (supported from v2.9.0). The interface can change in the future.
  return pickle.loads(x)
[I 2021-07-22 17:40:15,552] Trial 92 finished with value: 0.96 and parameters: {'max_depth': 9, 'n_estimators': 83}. Best is trial 84 with value: 0.9688888888888889.
[I 2021-07-22 17:40:15,555] Trial 93 finished with value: 0.9555555555555556 and parameters: {'max_depth': 9, 'n_estimators': 83}. Best is trial 84 with value: 0.9688888888888889.
[I 2021-07-22 17:40:15,602] Trial 94 finished with value: 0.9644444444444444 and parameters: {'max_depth': 9, 'n_estimators': 83}. Best is trial 84 with value: 0.9688888888888889.
[I 2021-07-22 17:40:15,677] Trial 95 finished with value: 0.9488888888888889 and parameters: {'max_depth': 9, 'n_estimators': 83}. Best is trial 84 with value: 0.9688888888888889.
[I 2021-07-22 17:40:15,816] Trial 97 finished with value: 0.96 and parameters: {'max_depth': 9, 'n_estimators': 83}. Best is trial 84 with value: 0.9688888888888889.
[I 2021-07-22 17:40:15,821] Trial 96 finished with value: 0.9644444444444444 and parameters: {'max_depth': 9, 'n_estimators': 83}. Best is trial 84 with value: 0.9688888888888889.
[I 2021-07-22 17:40:15,869] Trial 98 finished with value: 0.96 and parameters: {'max_depth': 9, 'n_estimators': 78}. Best is trial 84 with value: 0.9688888888888889.
[I 2021-07-22 17:40:15,892] Trial 99 finished with value: 0.9577777777777777 and parameters: {'max_depth': 9, 'n_estimators': 78}. Best is trial 84 with value: 0.9688888888888889.
Best params: {'max_depth': 10, 'n_estimators': 95}

```

</details>